.github/workflows/build-images.yml

```diff
@@ -0,0 +1,219 @@
+        with:
+          ref: ${{github.event.pull_request.head.ref}}
+          repository: ${{github.event.pull_request.head.repo.full_name}}
+      - id: git-versions
+        run: |
+          echo "::set-output name=git-version::$(git describe --tags --always)"
+      - id: build-name-image
+        run: |
+          echo "::set-output name=build-name-image::dashboard"
+      - id: build-name-file
+        run: |
+          echo "::set-output name=build-name-file::$(echo "${{steps.build-name-image.outputs.build-name-image}}" | tr '/' '-')"
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Build and export
+        run: |
+          make image-dashboard
+
+      - name: Login to Coding Container Registry
+        uses: docker/login-action@v1
+        with:
+          registry: zsnmwy-docker.pkg.coding.net
+          username: ${{ secrets.CODING_USERNAME }}
+          password: ${{ secrets.CODING_PASSWORD }}
+
+      - name: Push image
+        run: |
+          IMAGE_NAME=${{steps.build-name-image.outputs.build-name-image}}
+          IMAGE_ID=zsnmwy-docker.pkg.coding.net/gocrane/crane/$IMAGE_NAME
+
+          # Change all uppercase to lowercase
+          IMAGE_ID=$(echo $IMAGE_ID | tr '[A-Z]' '[a-z]')
+          VERSION=${{steps.git-versions.outputs.git-version}}
+          echo IMAGE_ID=$IMAGE_ID
+          echo VERSION=$VERSION
+          docker tag gocrane/$IMAGE_NAME:$VERSION $IMAGE_ID:pr-${{github.event.number}}-$VERSION
+          docker push $IMAGE_ID:pr-${{github.event.number}}-$VERSION
+
+  post-comment:
+    runs-on: ubuntu-latest
+    needs:
+      - build-craned
+      - build-metric-adapter
+      - build-crane-agent
+      - build-dashboard
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          ref: ${{github.event.pull_request.head.ref}}
+          repository: ${{github.event.pull_request.head.repo.full_name}}
+      - id: git-versions
+        run: |
+          echo "::set-output name=git-version::$(git describe --tags --always)"
+      - id: image-tags
+        run: |
+          echo "::set-output name=image-tags::pr-${{github.event.number}}-${{steps.git-versions.outputs.git-version}}"
+      - name: maintain-comment
+        uses: actions-cool/maintain-one-comment@v2
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          body: |
+            🎉 Successfully Build Images.
+
+            Overview: https://zsnmwy.coding.net/public-artifacts/gocrane/crane/packages
+            
+            | Image                                         | Pull Command                                              |
+            | --------------------------------------------- | --------------------------------------------------------- |
+            | crane-agent:${{steps.image-tags.outputs.image-tags}}    | docker pull zsnmwy-docker.pkg.coding.net/gocrane/crane/crane-agent:${{steps.image-tags.outputs.image-tags}}    |
```

Comment:
can this be put to crane space of coding, prefer not adding personal info in the target url